public/pages/CreateSnapshotPolicy/components/CronSchedule/CronSchedule.tsx

```diff
@@ -0,0 +1,199 @@
+    changeCron({ frequencyType });
+  }
+
+  const dayOfWeekCheckbox = (day: string, checkedDay: string) => (
+    <EuiFlexItem key={day} grow={false} style={{ marginRight: "0px" }}>
+      <EuiCheckbox id={day} label={day} checked={checkedDay === day} onChange={(e) => onDayOfWeekChange(day)} compressed />
+    </EuiFlexItem>
+  );
+
+  let additionalContent;
+  if (frequencyType === "weekly") {
+    // TODO SM if use dayOfWeek not initWeek, somehow it would be SUN
+    additionalContent = <EuiFlexGroup>{WEEK_DAYS.map((d) => dayOfWeekCheckbox(d, initWeek))}</EuiFlexGroup>;
+  }
+  if (frequencyType === "monthly") {
+    additionalContent = (
+      <>
+        <CustomLabel title="On the" />
+        <EuiFlexGroup gutterSize="m">
+          <EuiFlexItem>
+            <EuiSelect options={[{ value: "day", text: "Day" }]} defaultValue="Day" />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFieldNumber
+              value={dayOfMonth}
+              onChange={(e) => {
+                console.log(`sm dev change day of month ${parseInt(e.target.value)}`);
+                onDayOfMonthChange(parseInt(e.target.value));
+              }}
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </>
+    );
+  }
+
+  const cronExpressionHelpText = (
+    <EuiText color="subdued" size="s" style={{ padding: "5px 0px" }}>
+      <p style={{ fontWeight: 200, fontSize: "12px" }}>
+        Use Cron expression to define complex schedule.{" "}
+        <EuiLink href={CRON_EXPRESSION_DOCUMENTATION_URL} target="_blank">
+          Learn more
+        </EuiLink>
+      </p>
+    </EuiText>
+  );
+
+  return (
+    <>
+      <CustomLabel title="Schedule frequency" />
+      <EuiSelect id="creationCronScheduleType" options={CRON_SCHEDULE_FREQUENCY_TYPE} value={frequencyType} onChange={onTypeChange} />
+
+      <EuiSpacer size="m" />
+
+      <EuiFlexGroup justifyContent="flexStart" alignItems="flexStart">
+        <EuiFlexItem style={{ maxWidth: 400 }}>
+          {frequencyType === "custom" ? (
+            <>
+              <CustomLabel title="Cron expression" helpText={cronExpressionHelpText} />
+              <EuiFieldText
+                value={cronExpression}
+                onChange={(e) => {
+                  onCronExpressionChange(e.target.value);
+                }}
+              />
+            </>
+          ) : (
+            <>
+              <CustomLabel title="Start time" />
+              <EuiDatePicker
+                showTimeSelect
+                showTimeSelectOnly
+                selected={startTime(hour, minute)}
+                onChange={onStartTimeChange}
+                dateFormat="HH:mm"
+                timeFormat="HH:mm"
+              />
+              <EuiSpacer size="s" />
+
+              {additionalContent}
+            </>
+          )}
+        </EuiFlexItem>
+
+        {showTimezone ? (
+          <EuiFlexItem style={{ maxWidth: 250 }}>
+            <CustomLabel title="Time zone" />
```

Comment:
Minor observation. The `Timezone` field doesn't line up with the `Cron expression` field. 

![Screen Shot 2022-06-22 at 5 14 40 PM](https://user-images.githubusercontent.com/79280347/175181499-c09043b4-07ad-498c-8793-b996b78582a3.png)

Reply:
It's set to align at start which seems to the most stable one, timzone position won't change, but the left would change according to frequency type.